### DISCUSSION TITLE
Merging pdf's

### DISCUSSION THREAD

**Discussion by ju5t** [spatie/laravel-pdf#42] [Q&A]
A few weeks ago @freekmurze mentioned merging PDF's on X. We use fpdi for this in our own wrapper for Browsershot but I see no reason to keep it around as this package is much better.

I haven't found an elegant way for this at the moment (ours isn't). But before I send a PR in, are there any plans for this? What method is preferred to merge PDFs? Are there any other requirements (e.g. caching)?

**Comment by freekmurze** [comment]
There currently aren't plan for this. If the code to achieve this is simple and testable, there's a high chance I'll merge it.

**Comment by ahathaway** [comment]
@ju5t did anything ever come of this?
